test/Compatibility/unqualified_await_5.swift

```diff
@@ -0,0 +1,39 @@
+// RUN: %target-typecheck-verify-swift -swift-version 5
+    // now check that the fix-its, if applied, will fix the warnings.
+// RUN: %empty-directory(%t.scratch)
+// RUN: cp %s %t.scratch/fixits.swift
+// RUN: %target-swift-frontend -typecheck %t.scratch/fixits.swift -fixit-all -emit-fixits-path %t.scratch/fixits.remap 2> /dev/null
+// RUN: %{python} %utils/apply-fixit-edits.py --one-fixit-each %t.scratch
+// RUN: %target-swift-frontend -typecheck %t.scratch/fixits.swift -warnings-as-errors -swift-version 5
+// RUN: %target-swift-frontend -typecheck %t.scratch/fixits.swift -warnings-as-errors -enable-experimental-concurrency
+
+func await(_ f : () -> Void) {}
```

Comment:
Sorry for the random comment, but I was just wondering if this also worth a warning too like `future versions of Swift reserve the word 'await'; if this name is unavoidable, use backticks to escape it`. Because once this becomes a reserved word keyword, use this as an identifier would be allowed only with backticks e.g.` func 'await'(_ f : () -> Void) {}` same way as other keywords ... so if that's the case maybe would make sense as well :)

Reply:
Thanks, good point here. I think instead of a fix-it to fully-qualify the name, we'll instead suggest using backticks at the use-sites, unless if it's a module-private declaration, where we'll instead suggest changing the name.

The implementation of async/await does not require that declarations named `await` use a backticks, only the use-sites of the declaration, because `await` is considered a keyword only in contexts that are expressions.

Reply:
> The implementation of async/await does not require that declarations named `await` use a backticks, only the use-sites of the declaration, because `await` is considered a keyword only in contexts that are expressions.

Ah I see, thanks! I thought this was a general rule for all keywords independently of context, so that is good to know 

> I think instead of a fix-it to fully-qualify the name, we'll instead suggest using backticks at the use-sites, unless if it's a module-private declaration, where we'll instead suggest changing the name. 

That looks like a good approach :)